electron/src/main.ts

```diff
@@ -363,7 +363,7 @@ const handleAppEvents = () => {
             logger.log('Proxy prompt was submitted');
 
             const {username, password} = promptData;
-            const protocol: string | undefined = proxyInfoArg?.protocol?.replace(':', '');
+            const protocol = proxyInfoArg?.protocol?.replace(':', '');
```

Comment:
What's the intention here? Can you comment it or have a test for it?

Reply:
The intention was to remove useless TypeScript annotations.

Reply:
I don't think that the intention of `replace(':', '')` is to remove useless TypeScript annotations or is it?

Reply:
What do you mean? This line was added [in #3634](https://github.com/wireapp/wire-desktop/commit/dda822ed31fe636a4a70205fad898430a2ba9ee0#diff-acab454f418662fe6f697507e4c4103aR367), I just removed the type annotations.